pkg/resources/management.cattle.io/v3/globalrolebinding/validator.go

```diff
@@ -37,13 +38,17 @@ var (
 const bindVerb = "bind"
 
 // NewValidator returns a new validator for GlobalRoleBindings.
-func NewValidator(resolver rbacvalidation.AuthorizationRuleResolver, grbResolver *resolvers.GRBClusterRuleResolver,
-	sar authorizationv1.SubjectAccessReviewInterface) *Validator {
+func NewValidator(resolver rbacvalidation.AuthorizationRuleResolver, icrResolver *resolvers.GRBClusterRuleResolver,
```

Comment:
Similar comment here about making the parameter order at minimum commented, and probably also looking into some better u/x here.

Reply:
see https://github.com/rancher/webhook/pull/348#discussion_r1576015559